modules/sharethroughBidAdapter.js

```diff
@@ -1,128 +1,71 @@
-var utils = require('src/utils.js');
-var bidmanager = require('src/bidmanager.js');
-var bidfactory = require('src/bidfactory.js');
-var ajax = require('src/ajax.js').ajax;
-var adaptermanager = require('src/adaptermanager');
-
-const STR_BIDDER_CODE = 'sharethrough';
-const STR_VERSION = '1.2.0';
-
-var SharethroughAdapter = function SharethroughAdapter() {
-  const str = {};
-  str.STR_BTLR_HOST = document.location.protocol + '//btlr.sharethrough.com';
-  str.STR_BEACON_HOST = document.location.protocol + '//b.sharethrough.com/butler?';
-  str.placementCodeSet = {};
-  str.ajax = ajax;
-
-  function _callBids(params) {
-    const bids = params.bids;
-
-    // cycle through bids
-    for (let i = 0; i < bids.length; i += 1) {
-      const bidRequest = bids[i];
-      str.placementCodeSet[bidRequest.placementCode] = bidRequest;
-      const scriptUrl = _buildSharethroughCall(bidRequest);
-      str.ajax(scriptUrl, _createCallback(bidRequest), undefined, {withCredentials: true});
-    }
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'sharethrough';
+const VERSION = '2.0.0';
+const STR_ENDPOINT = document.location.protocol + '//btlr.sharethrough.com/header-bid/v1';
+
+export const sharethroughAdapterSpec = {
+  code: BIDDER_CODE,
+  isBidRequestValid: bid => !!bid.params.pkey && bid.bidder === BIDDER_CODE,
+  buildRequests: (bidRequests) => {
+    return bidRequests.map(bid => {
+      return {
+        method: 'GET',
+        url: STR_ENDPOINT,
+        data: {
+          bidId: bid.bidId,
+          placement_key: bid.params.pkey,
+          hbVersion: '$prebid.version$',
+          strVersion: VERSION,
+          hbSource: 'prebid'
+        }
+      };
+    })
+  },
+  interpretResponse: (resp, req) => {
```

Comment:
#1748 changed the first argument of `interpretResponse` to:

```JavaScript
{
  body: responseBody,
  headers: {
    get: function(header) { /* returns a header from the HTTP response */ }
  }
}
```

so adding something like
```JavaScript
resp = resp.body;
```
just below this line, or however you'd prefer to grab the `body`, and updating corresponding tests if needed should get this back to working properly